### PR TITLE
Description for imageset api

### PR DESCRIPTION
**Pull request by mgold1234**
# Description
Description for imageset api

FIXES: related to epic https://issues.redhat.com/browse/THEEDGE-3383

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Documentation update
- [ ] Tests update
- [x] Refactor

### CODE REVIEW

File: pkg/models/imagesets_api.go [hunk 1]
```diff
@@ -2,59 +2,60 @@ package models
...
 type ImageSetAPI struct {
 	Model
-	Name    string  `json:"name" example:"my-edge-image"`
-	Version int     `json:"version" example:"1"`
-	Account string  `json:"Account" example:"0000"`
-	Images  []Image `json:"Images"`
+	Name    string  `json:"name" example:"my-edge-image"`      // the image set name
+	Version int     `json:"version" example:"1"`               // the image set version
+	Account string  `json:"Account" example:"001100"`          // the account that image set created from
+	Images  []Image `json:"Images" example:"[image1, image2]"` // images of image set
```
Comment (ldjebran) [inline pkg/models/imagesets_api.go:8]: remove the example

File: pkg/models/imagesets_api.go [hunk 2]
```diff
@@ -2,59 +2,60 @@ package models
...
 }
 
 // ImageSetIDViewAPI is the image set details view returned for ui image-set display
 type ImageSetIDViewAPI struct {
-	ImageBuildIsoURL string         `json:"ImageBuildIsoURL"`
-	ImageSet         ImageSetAPI    `json:"ImageSet"`
-	LastImageDetails ImageDetailAPI `json:"LastImageDetails"`
+	ImageBuildIsoURL string         `json:"ImageBuildIsoURL" example:"/api/edge/v1/storage/isos/432"` // image build ISO url
+	ImageSet         ImageSetAPI    `json:"ImageSet"`                                                 // image set data
+	LastImageDetails ImageDetailAPI `json:"LastImageDetails"`                                         // last image detail
```
Comment (ldjebran) [inline pkg/models/imagesets_api.go:28]: // The image-set latest image details

File: pkg/models/imagesets_api.go [hunk 3]
```diff
@@ -2,59 +2,60 @@ package models
...
+	Count int            `json:"count" example:"10"` // count of image-sets
+	Data  []ImageSetView `json:"data"`               // data of image set view
 }
 
 // ImageSetIDViewAPI is the image set details view returned for ui image-set display
 type ImageSetIDViewAPI struct {
-	ImageBuildIsoURL string         `json:"ImageBuildIsoURL"`
-	ImageSet         ImageSetAPI    `json:"ImageSet"`
-	LastImageDetails ImageDetailAPI `json:"LastImageDetails"`
+	ImageBuildIsoURL string         `json:"ImageBuildIsoURL" example:"/api/edge/v1/storage/isos/432"` // image build ISO url
```
Comment (ldjebran) [inline pkg/models/imagesets_api.go:26]: // The latest available image ISO artifact

File: pkg/models/imagesets_api.go [hunk 4]
```diff
@@ -2,59 +2,60 @@ package models
...
+	ImageSet         ImageSetAPI    `json:"ImageSet"`                                                 // image set data
+	LastImageDetails ImageDetailAPI `json:"LastImageDetails"`                                         // last image detail
 }
 
 // ImageSetImageIDViewAPI is the image set image view returned for ui image-set / version display
 type ImageSetImageIDViewAPI struct {
-	ImageBuildIsoURL string         `json:"ImageBuildIsoURL"`
-	ImageSet         ImageSetAPI    `json:"ImageSet"`
-	ImageDetails     ImageDetailAPI `json:"ImageDetails"`
+	ImageBuildIsoURL string         `json:"ImageBuildIsoURL" example:"/api/edge/v1/storage/isos/432"` // image build ISO url
```
Comment (ldjebran) [inline pkg/models/imagesets_api.go:33]: // The image-set latest available image ISO

File: pkg/models/imagesets_api.go [hunk 5]
```diff
@@ -2,59 +2,60 @@ package models
...
+	Count int      `json:"Count" example:"10"` // count of image-set's devices
+	Data  []string `json:"Data"`               // Data of image set's devices
 }
 
 // ImageSetInstallerURLAPI returns ImagesetAPI structure with last installer available
 type ImageSetInstallerURLAPI struct {
-	ImageSetData     ImageSetAPI `json:"image_set"`
-	ImageBuildISOURL *string     `json:"image_build_iso_url" example:"https://buket.example.com"`
+	ImageSetData     ImageSetAPI `json:"image_set"`                                                   // image set data
+	ImageBuildISOURL *string     `json:"image_build_iso_url" example:"/api/edge/v1/storage/isos/432"` // image build ISO url
```
Comment (ldjebran) [inline pkg/models/imagesets_api.go:47]: // The image-set latest available image ISO

File: pkg/models/imagesets_api.go [hunk 6]
```diff
@@ -2,59 +2,60 @@ package models
...
 }
 
 // ImageSetImagePackagesAPI return info related to details on images from imageset
 type ImageSetImagePackagesAPI struct {
-	ImageSetData     ImageSetAPI      `json:"image_set"`
-	Images           []ImageDetailAPI `json:"images"`
-	ImageBuildISOURL string           `json:"image_build_iso_url"`
+	ImageSetData     ImageSetAPI      `json:"image_set"`                                                   // image set data
+	Images           []ImageDetailAPI `json:"images"`                                                      // image detail
+	ImageBuildISOURL string           `json:"image_build_iso_url" example:"/api/edge/v1/storage/isos/432"` // image build ISO url
```
Comment (ldjebran) [inline pkg/models/imagesets_api.go:15]: // The image-set latest available image ISO

**Review by ldjebran — COMMENTED** [review]
reviewed imagesets_api.go only as part of this issues (other changes are part of other PR)
Overall looks good , left some comments.

**Review by adeliaferreira — APPROVED** [review]
lgtm

**Pull request merged**